documentation-site/components/yard/config/checkbox.ts

```diff
@@ -0,0 +1,185 @@
+import {Checkbox, STYLE_TYPE, LABEL_PLACEMENT} from 'baseui/checkbox';
+import {PropTypes} from '../const';
+
+export default {
+  scopeConfig: {
+    Checkbox,
+    STYLE_TYPE,
+    LABEL_PLACEMENT,
+  },
+  themeConfig: [
+    'tickFill',
+    'tickFillHover',
+    'tickFillActive',
+    'tickFillSelected',
+    'tickFillSelectedHover',
+    'tickFillSelectedHoverActive',
+    'tickFillError',
+    'tickFillErrorHover',
+    'tickFillErrorHoverActive',
+    'tickFillErrorSelected',
+    'tickFillErrorSelectedHover',
+    'tickFillErrorSelectedHoverActive',
+    'tickFillDisabled',
+    'tickBorder',
+    'tickBorderError',
+    'tickMarkFill',
+    'tickMarkFillError',
+    'tickMarkFillDisabled',
+  ],
+  propsConfig: {
+    checked: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'Renders component in checked state.',
+      meta: {
+        stateful: true,
+      },
+    },
+    children: {
+      value: `Sign up for the newsletter`,
+      type: PropTypes.ReactNode,
+      description: `The React Nodes displayed next to the checkbox.`,
+    },
+    disabled: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'Renders component in disabled state.',
+    },
+    checkmarkType: {
+      value: undefined,
+      options: STYLE_TYPE,
+      type: PropTypes.Enum,
+      enumName: 'STYLE_TYPE',
+      description: 'Renders component in a toggle state.',
+    },
+    onChange: {
+      value: 'e => setChecked(e.target.checked)',
+      type: PropTypes.Function,
+      description: 'Called when checkbox value is changed.',
+      meta: {
+        propHook: {
+          what: 'e.target.checked',
+          into: 'checked',
+        },
+      },
+    },
+    isError: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'Renders component in error state.',
+    },
+    isIndeterminate: {
+      value: false,
+      type: PropTypes.Boolean,
+      description:
+        'Indicates indeterminate state for the checkmark. Checked property is ignored.',
+    },
+    labelPlacement: {
+      value: undefined,
+      options: LABEL_PLACEMENT,
+      type: PropTypes.Enum,
+      enumName: 'LABEL_PLACEMENT',
+      description:
+        'Determines how to position the label relative to the checkbox.',
+    },
+    //////////////////
+    // Hidden props //
+    //////////////////
+    required: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'Renders component in required state.',
+      hidden: true,
+    },
+    inputRef: {
+      value: undefined,
+      type: PropTypes.Ref,
+      description: 'A ref to access an input element.',
+      hidden: true,
+    },
+    autoFocus: {
+      value: false,
+      type: PropTypes.Boolean,
+      description: 'If true the component will be focused on the first mount.',
+      hidden: true,
+    },
+    type: {
```

Comment:
hm, what is this prop for? should it always be "checkbox"?

at least I would set `hidden: true`, doesn't seem important

Reply:
yeah, the default value for this is checkbox - I'll just remove this from yard